src/component/framework/association.js

```diff
@@ -5,6 +5,8 @@ export class Association {
 
   association;
   manyAssociation;
+  selectablePlaceholder = false;
+  placeholderValue      = 0;
```

Comment:
These two values are never used.

Perhaps you meant to use them as default values in `activate()` ([line 30](https://github.com/SpoonX/aurelia-form/pull/104/files#diff-ad697cea9b9f791e98207d3b65d56cc0R30))?

```js
element.placeholderValue = this.placeholderValue || 0
```